### PR TITLE
fix: Text Style on Flutter 3.22

### PR DESCRIPTION
**Pull request by chandrabezzo**
# Description

After Flutter SDK 3.22, removed deprecated text theme for headline6 and subtitle2. To fix this issue I follow migration guide from [here](https://docs.flutter.dev/release/breaking-changes/3-19-deprecations#texttheme).

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issue
#434 

## Result
Screenshot 2024-05-14 at 09 07 13

Follow Migration Guide
Screenshot 2024-05-15 at 08 19 08

[Flutter Release Notes 3.22]: https://docs.flutter.dev/release/release-notes/release-notes-3.22.0

### CODE REVIEW

**Review by gmWheeler001 — APPROVED** [review]

**Comment by VB10** [comment]
lgtm 🌸

**Review by VB10 — APPROVED** [review]

**Review by ashu3699 — APPROVED** [review]

**Review by ashu3699 — CHANGES_REQUESTED** [review]
please add this..

**Review by ashu3699 — APPROVED** [review]

**Review by JoseDRojasA — APPROVED** [review]

**Review by ziqq — APPROVED** [review]

**Comment by aditya-css** [comment]
Hello @chandrabezzo, Thank you for raising this PR. Can you please bump the version to 3.0.0 pubspec and changelog?

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: showcaseview
 description: A Flutter package to Showcase/Highlight widgets step by step.
-version: 2.1.1
+version: 2.1.2
```
Comment (aditya-css) [inline pubspec.yaml:3]: Please update this to 3.0.0.

**Review by aditya-css — CHANGES_REQUESTED** [review]

**Comment by chandrabezzo** [comment]
Done @aditya-css

**Review by aditya-css — APPROVED** [review]

**Comment by Obada2020** [comment]
how to pull this in my git

**Comment by victor337** [comment]
> how to pull this in my git

I think he will update in pub.dev, but in the meantime use this:

```
showcaseview:
    git:
      url: https://github.com/chandrabezzo/flutter_showcaseview
      ref: ae8d8a327cdede529d264d1df02892fdc6161688
```

**Pull request merged**